lib/utilities/positionUtils.js

```diff
@@ -0,0 +1,114 @@
+var CardinalDirection = require('../enum/cardinalDirection');
+var CardinalPortPosition = require('../model/cardinalPortPosition');
+var CartesianPortPosition = require('../model/cartesianPortPosition');
+/**
+ * @class PositionUtils
+ * @static
+ */
+var PositionUtils = {};
+
+// Data model scales values up to 0-100.
+var DATA_MODEL_MULTIPLIER = 100.0;
```

Comment:
I don't think that you need to add the `.0` to the end. JS has only one number type - `int` and `float` are not distinct.